lib/classes/Utils.js

```diff
@@ -206,9 +207,9 @@ class Utils {
               // For HTTP events, see what authorizer types are enabled
               if (event.http && event.http.authorizer) {
                 if ((typeof event.http.authorizer === 'string'
```

Comment:
We should use lodash consistently and change that probably right away in this file:
`_.has(event, 'http.authorizer') && _.isString(event.http.authorizer)`